product/views.py

```diff
@@ -1,8 +1,42 @@
-
-
-
-
-
-
-
-
+import json
+
+from django.views   import View
+from django.http    import JsonResponse
+
+from product.models import Drink, SubCategory, Nutrition
+
+class DrinkListView(View):
+    def get(self, request):
+        names            = request.GET.getlist("name", None)
+        filter_set       = {}
+
+        if names:
+            filter_set['name__in'] = names
+            
+        subcategory_list = [
+        {
+        'subcategory_id' : subcategory.id,
+        'name'           : subcategory.name,
+        'description'    : subcategory.description,
+        'products' : [
+            {
+                'id'         : drink.id,
+                'isnew'      : drink.is_new,
+                'isseason'   : drink.is_season,
+                'title'      : drink.korean_name,
+                'image'      : drink.image_url,
+                'size'       : drink.nutritions.all()[0].size.name,
+                'kcal'       : drink.nutritions.all()[0].kcal,
+                'sodium'     : drink.nutritions.all()[0].sodium,
+                'saturation' : drink.nutritions.all()[0].saturation,
+                'sugar'      : drink.nutritions.all()[0].sugar,
+                'protein'    : drink.nutritions.all()[0].protein,
+                'caffeine'   : drink.nutritions.all()[0].caffeine
```

Comment:
```suggestion
                'size'       : drink.nutritions.first().size.name,
                'kcal'       : drink.nutritions.first().kcal,
                'sodium'     : drink.nutritions.first().sodium,
                'saturation' : drink.nutritions.all()[0].saturation,
                'sugar'      : drink.nutritions.all()[0].sugar,
                'protein'    : drink.nutritions.all()[0].protein,
                'caffeine'   : drink.nutritions.all()[0].caffeine
```
drink의 nutrition 속성이 한세트인 걸로 이해했는데 맞나요?
어차피 하나 밖에 없는 속성이라면 `first()` 메소드를 한 번 활용해보시죠! :)